human_general_analysis/nbconverted/Try_other_enrichment_methods.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Name
+# 
+# This notebook plugs in other gene set enrichment methods to demonstrate that our method, SOPHIE, can be inserted into different pipelines and work with other methods
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('load_ext', 'rpy2.ipython')
+get_ipython().run_line_magic('autoreload', '2')
+
+import os
+import sys
+import pandas as pd
+import numpy as np
+import pickle
+
+from rpy2.robjects import pandas2ri
+pandas2ri.activate()
+
+from ponyo import utils
```

Comment:
What tensorflow version are you using? If it is less than 2.x.x, your ponyo code might become outdated. ([Example Here](https://github.com/KrishnaswamyLab/SAUCIE/pull/38) Really tiny issue in the overall scheme, but I wanted to just let you know.

Reply:
Thanks for looking out. I remember having some conflicts with other dependencies that arose when i previously tried to upgrade tensorflow. But it has been a while since I looked at this so I'll make an issue in my ponyo repo, which is using v<2.x.x for tensorflow